Juliana/second-assignment/script.js

```diff
@@ -0,0 +1,68 @@
+const nav = document.querySelector("#nav");
+const open = document.querySelector("#open");
+const close = document.querySelector("#close");
+const loadHeroesBtn = document.querySelector('#loadHeroes');
+const loadSortedHeroesBtn = document.querySelector('#loadSortedHeroes');
+const heroesContainer = document.getElementById('heroes-list');
+const sortedHeroesContainer = document.getElementById('sorted-heroes-list');
+let arrHeroes = [];
+
+open.addEventListener("click", () => nav.classList.add("visible"));
+close.addEventListener("click", () => nav.classList.remove("visible"));
+
+loadHeroesBtn.addEventListener('click', () => {
+    replaceContainer(sortedHeroesContainer, heroesContainer);
+    document.querySelector('#loadHeroes').style.display = 'none';
+    document.querySelector('#loadSortedHeroes').style.display = 'flex';    
+});
+loadSortedHeroesBtn.addEventListener('click', () => {
+    replaceContainer(heroesContainer, sortedHeroesContainer);
+    sortHeroes();
+    showHeroes(arrSortedHeroes, sortedHeroesContainer);
+    document.querySelector('#loadHeroes').style.display = 'flex';
+    document.querySelector('#loadSortedHeroes').style.display = 'none';
+});
+
+function showData(){
+    const url = './marvel.json';
```

Comment:
Sorry @JulianaCotabarren, I've just noticed that you are using the JSON inside your folder, please use the one in the root mock folder so we don't have duplicate files.

Reply:
Done in commit [ac3270d]